.envrc

```diff
@@ -0,0 +1 @@
+use_flake
```

Comment:
When including a flake.nix in this repo, it would be probably better to just use `use flake` (cf. https://github.com/direnv/direnv/blob/master/man/direnv-stdlib.1.md#use-flake-installable)